nautobot/dcim/models/device_components.py

```diff
@@ -773,6 +778,87 @@ def parent(self):
         return self.device
 
 
+@extras_features(
+    "custom_fields",
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "relationships",
+    "statuses",
+    "webhooks",
+)
+class InterfaceRedundancyGroup(PrimaryModel, ConfigContextModel, StatusModel):  # pylint: disable=too-many-ancestors
+    """
+    A collection of Interfaces that supply a redundancy group for protocols like HSRP/VRRP.
+    """
+
+    name = models.CharField(max_length=100, unique=True)
+    slug = AutoSlugField(populate_from="name")
+    description = models.CharField(max_length=200, blank=True)
+    virtual_ip = models.ForeignKey(
+        to="ipam.IPAddress",
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="interface_redundancy_virtual_ip",
+    )
+    interfaces = models.ManyToManyField(
+        to="dcim.Interface",
+        through="dcim.InterfaceRedundancyGroupAssociation",
+        related_name="redundancy_groups",
```

Comment:
For consistency this should also be `interface_redundancy_groups` in keeping with the `verbose_name_plural` of the `InterfaceRedundancyGroup` model.

Reply:
fixed